docs/docs/actions.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Actions
+jsdoc:
+  - "gatsby/src/redux/actions/public.js"
+  - "gatsby/src/redux/actions/restricted.js"
+contentsHeading: Functions
+---
+
+Gatsby uses [Redux](http://redux.js.org) internally to manage state. When you implement a Gatsby API, you are passed a collection of actions (equivalent to [bindActionCreators](https://redux.js.org/api/bindactioncreators/) in Redux) which you can use to manipulate state on your site.
```

Comment:
We currently have note in parenthesis worded as

> equivalent to actions bound with bindActionCreators in redux

from https://github.com/gatsbyjs/gatsby/blob/8c003325cd9632683107d04bdf807dec28a51b62/www/src/pages/docs/actions.js#L43-L49

which to me has a bit different meaning than what's in this change:

> equivalent to bindActionCreators in Redux

Let's not make any content changes in this PR:

```suggestion
Gatsby uses [Redux](http://redux.js.org) internally to manage state. When you implement a Gatsby API, you are passed a collection of actions (equivalent to actions bound with [bindActionCreators](https://redux.js.org/api/bindactioncreators/) in Redux) which you can use to manipulate state on your site.
```